telemetry/service/service-model.json

```diff
@@ -64,6 +64,21 @@
       "type":"string",
       "max":2000
     },
+    "CommputeEnv":{
+      "type":"string",
+      "enum":[
+        "cloud9",
+        "codeCatalyst",
```

Comment:
do we need a type for a Cloud9 + CodeCatalyst?

Reply:
since this is (currently) an enum instead of a list, yes. there should be a `cloud9Codecatalyst` item

Reply:
what are other combinations? if env can be combined & there are multiple combinations, it is good idea to instrument on Tookits' code insead?

Reply:
> what are other combinations? if env can be combined

That's a judgment call. The idea here (and in the quip doc) is to decide out. I think we have decided it by now. The combinations are as we've listed them here (after making the changes mentioned in this PR).

Reply:
got it.